### PR TITLE
Add EI Analytics feature

### PR DESCRIPTION
**Pull request by isharailanga**
## Purpose
This PR is to add ei-solution-feature for EI-Analytics

### CODE REVIEW

File: features/ei-solution-features/org.wso2.carbon.ei.solutions.ei.analytics.feature/pom.xml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    <name>WSO2 Stream Processor - EI Analytics Solution Feature</name>
+    <artifactId>org.wso2.carbon.ei.solutions.ei.analytics.feature</artifactId>
+
+
+    <dependencies>
+        <!--Widgets-->
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.carbon.analytics.solutions.ei.analytics.horizontal.bar.chart</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```
Comment (dilini-muthumala) [inline features/ei-solution-features/org.wso2.carbon.ei.solutions.ei.analytics.feature/pom.xml]: Please remove version tag.

File: features/ei-solution-features/org.wso2.carbon.ei.solutions.ei.analytics.feature/pom.xml [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.carbon.analytics.solutions.ei.analytics.mediator.properties</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```
Comment (dilini-muthumala) [inline features/ei-solution-features/org.wso2.carbon.ei.solutions.ei.analytics.feature/pom.xml]: Please remove version tag. Version should be coming from the project pom file. Kindly fix in other places as well (in this pom file).

File: components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingList/pom.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.analytics.solutions.sp.message.tracer.widgets.open-tracing-list</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (dilini-muthumala) [inline components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingList/pom.xml]: Please remove this version element.

File: components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingSearch/pom.xml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.analytics.solutions.sp.message.tracer.widgets.open-tracing-search</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (dilini-muthumala) [inline components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingSearch/pom.xml]: Please remove version element.

File: components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingVisTimeline/pom.xml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.analytics.solutions.sp.message.tracer.widgets.open-tracing-vis-timeline</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (dilini-muthumala) [inline components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingVisTimeline/pom.xml]: Please remove this version element, as it is inherited from parent tag.

File: components/sp-solutions/org.wso2.analytics.solutions.sp.twitter.analytics/widgets/Emotions/pom.xml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.analytics.solutions.sp.twitter.analytics.widgets.emotions</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (dilini-muthumala) [inline components/sp-solutions/org.wso2.analytics.solutions.sp.twitter.analytics/widgets/Emotions/pom.xml]: Please remove this version element.

File: features/ei-solution-features/org.wso2.carbon.ei.solutions.ei.analytics.feature/pom.xml (hunk 1 above)
Comment (isharailanga) [reply features/ei-solution-features/org.wso2.carbon.ei.solutions.ei.analytics.feature/pom.xml]: Fixed in d0663ee31e3530ba950220f24e7c317eb333d7c8

File: features/ei-solution-features/org.wso2.carbon.ei.solutions.ei.analytics.feature/pom.xml (hunk 2 above)
Comment (isharailanga) [reply features/ei-solution-features/org.wso2.carbon.ei.solutions.ei.analytics.feature/pom.xml]: Fixed in d0663ee31e3530ba950220f24e7c317eb333d7c8

File: components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingList/pom.xml (hunk 3 above)
Comment (isharailanga) [reply components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingList/pom.xml]: Fixed in d0663ee31e3530ba950220f24e7c317eb333d7c8

File: components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingSearch/pom.xml (hunk 4 above)
Comment (isharailanga) [reply components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingSearch/pom.xml]: Fixed in d0663ee31e3530ba950220f24e7c317eb333d7c8

File: components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingVisTimeline/pom.xml (hunk 5 above)
Comment (isharailanga) [reply components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingVisTimeline/pom.xml]: Fixed in d0663ee31e3530ba950220f24e7c317eb333d7c8

Comment (isharailanga) [reply components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingVisTimeline/pom.xml]: Fixed in d0663ee31e3530ba950220f24e7c317eb333d7c8

File: components/sp-solutions/org.wso2.analytics.solutions.sp.twitter.analytics/widgets/Emotions/pom.xml (hunk 6 above)
Comment (isharailanga) [reply components/sp-solutions/org.wso2.analytics.solutions.sp.twitter.analytics/widgets/Emotions/pom.xml]: Fixed in d0663ee31e3530ba950220f24e7c317eb333d7c8

**Pull request merged**